src/improvements/tasks/sep/007-opcm-upgrade-v300-uni/VALIDATION.md

```diff
@@ -0,0 +1,203 @@
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+  ---
+
+### [`0x0d83dab629f0e0f9d36c0cbc89b69a489f0751bd`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/superchain/configs/sepolia/unichain.toml#L59)  (OptimismPortal2) - Chain ID: 1301
+
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x2D7e764a0D9919e16983a46595CfA81fc34fa7Cd`
+  - **After:** [`0xB443Da3e07052204A02d630a8933dAc05a0d6fB4`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/validation/standard/standard-versions-sepolia.toml#L13)
+  - **Summary:**           ERC-1967 implementation slot
+  - **Detail:**            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ---
+
+### [`0x4696b5e042755103fe558738bcd1ecee7a45ebfe`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/superchain/configs/sepolia/unichain.toml#L56)  (L1ERC721Bridge) - Chain ID: 1301
+
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x276d3730f219f7ec22274f7263180b8452B46d47`
+  - **After:** [`0x7aE1d3BD877a4C5CA257404ce26BE93A02C98013`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/validation/standard/standard-versions-sepolia.toml#L19)
+  - **Summary:**           ERC-1967 implementation slot
+  - **Detail:**            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ---
+
+### [`0xaee94b9ab7752d3f7704bde212c0c6a0b701571d`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/superchain/configs/sepolia/unichain.toml#L60)  (SystemConfig) - Chain ID: 1301
+
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x760C48C62A85045A6B69f07F4a9f22868659CbCc`
+  - **After:** [`0x340f923E5c7cbB2171146f64169EC9d5a9FfE647`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/validation/standard/standard-versions-sepolia.toml#L28)
```

Comment:
I think you want this link: https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/validation/standard/standard-versions-sepolia.toml#L9